local-volume/provisioner/pkg/common/common.go

```diff
@@ -102,3 +110,49 @@ func CreateLocalPVSpec(config *LocalPVConfig) *v1.PersistentVolume {
 		},
 	}
 }
+
+// GetVolumeConfigFromConfigMap gets volume configuration from given configmap,
+func GetVolumeConfigFromConfigMap(client *kubernetes.Clientset, namespace, name string) (map[string]MountConfig, error) {
+	configMap, err := client.CoreV1().ConfigMaps(namespace).Get(name, metav1.GetOptions{})
+	if err != nil {
+		return nil, err
+	}
+	return ConfigMapDataToVolumeConfig(configMap.Data)
+}
+
+// GetDefaultVolumeConfig returns the default volume configuration.
+func GetDefaultVolumeConfig() map[string]MountConfig {
+	return map[string]MountConfig{
+		"local-storage": {
+			HostDir:  DefaultHostDir,
+			MountDir: DefaultMountDir,
+		},
+	}
+}
+
+// VolumeConfigToConfigMapData converts volume config to configmap data.
+func VolumeConfigToConfigMapData(config map[string]MountConfig) (map[string]string, error) {
+	configMapData := make(map[string]string)
+	for class, data := range config {
```

Comment:
Is it possible to just marshal/unmarshal the entire map in one call, so you don't have to iterate through the map?